docs/docs/0.7.1/documentation/dps-and-data-model.md

```diff
@@ -89,17 +84,22 @@ networking:
   outbound:
   - port: 10000
     tags:
-      kuma.io/service: redis" | kumactl apply -f -
+      kuma.io/service: redis
 
 kuma-dp run \
-  --name=backend-1 \
-  --mesh=default \
   --cp-address=http://127.0.0.1:5681 \
+  --dataplane-file=dp.yaml \
+  --dataplane-var name=`hostname -s` \
+  --dataplane-var address=192.168.0.2 \
   --dataplane-token-file=/tmp/kuma-dp-backend-1-token
 ```
 
 In order for the `backend` service to successfully consume `redis`, we specify an `outbound` networking section in the `Dataplane` configuration instructing the DP to listen on a new port `10000` and to proxy any outgoing request on port `10000` to the `redis` service. For this to work, we must update our application to consume `redis` on `127.0.0.1:10000`.
 
+::: tip
+You can parametrize your `Dataplane` definition, so you can reuse the same file for many `kuma-dp` instances or even services.
```

Comment:
Shall we give examples, where one parametrizes port(s), tag(s) etc? So that users can see how flexible this approach is and adapt the workflow for their needs.

Reply:
I gave an example above that parametrizes name and mesh. Should I also parametrize port and tag?

Reply:
I think - yes. Just to give users a hint of how far they can go with the parameters.